packages/lesswrong/components/sunshineDashboard/SunshineNewUsersProfileInfo.tsx

```diff
@@ -32,7 +32,7 @@ const SunshineNewUsersProfileInfo = ({userId, classes}:{userId:string, classes:
   
   if (user.reviewedByUserId && !user.snoozedUntilContentCount && !expanded) {
     return <div className={classes.root} onClick={() => setExpanded(true)}>
-      <SectionButton>Expand</SectionButton>
+      <SectionButton>Expand Moderation Tools</SectionButton>
```

Comment:
Maybe?

```suggestion
      <SectionButton>{preferredHeadingCase("Expand Moderation Tools")}</SectionButton>
```

<img width="305" alt="Screenshot 2024-01-25 at 21 22 23" src="https://github.com/ForumMagnum/ForumMagnum/assets/5075628/9664b917-b7ff-4171-aaf4-0849580ebe9c">